sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/ServiceBusEndToEndTests.cs

```diff
@@ -343,6 +343,19 @@ public async Task TestSingle_ReceiveFromFunction()
             }
         }
 
+        [Test]
+        public async Task TestSingle_ReceiveFromDeadLetterQueue()
```

Comment:
The DLQ issue reported in the issue was a red herring, I believe. But adding a test for DLQ anyway.